.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "bracketSpacing": false,
+  "singleQuote": true,
```

Comment:
Let's not mix adding Prettier with typings. Could you please open a new PR adding Prettier to our workflow (including `npm test` executed by CI)? Alternatively, keep these prettier-related files local only to help your IDE, don't commit them to git.